### PR TITLE
fixing pairwise aligner tests for python3

### PR DESCRIPTION
**Pull request by mdehoon**
This pull request updates the unittest for the pairwise aligner to python3.

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Tests/test_pairwise_aligner.py [hunk 1]
```diff
@@ -1849,51 +1848,45 @@ class TestArgumentErrors(unittest.TestCase):
...
-                         str(context_manager.exception))
-        with self.assertRaises(ValueError) as context_manager:
+        message = "^sequence contains letters not in the alphabet$"
+        with self.assertRaisesRegex(ValueError, message):
             aligner.score("AAA", "AA&")
-        self.assertEqual("sequence contains letters not in the alphabet",
-                         str(context_manager.exception))
 
     def test_aligner_array_errors(self):
+        import array
```
Comment (peterjc) [inline Tests/test_pairwise_aligner.py]: Move the import to the top of the file?

**Comment by peterjc** [comment]
Why do you prefer defining ``message = ...`` as a local variable?

**Comment by mdehoon** [comment]
> Why do you prefer defining `message = ...` as a local variable?

To make travis happy.

**Comment by peterjc** [comment]
There's probably a better way to make Travis happy, but if you're happy with this, that's fine.

**Pull request merged**